common/app/views/fragments/headTonal.scala.html

```diff
@@ -13,7 +13,13 @@
                 @fragments.meta.metaInline(content)
 
                 <h1 class="content__headline js-score" itemprop="headline">
-                    @Html(content.headline)
+                    @content match {
+                        case v: Video => {
+                            @fragments.inlineSvg("video-icon", "icon")
+                            @Html(content.headline)
+                        }
+                        case _ => { @Html(content.headline) }
+                    }
```

Comment:
Using a case match for this because in a later PR will add Audio and Gallery types.